data/json/monsters/turrets.json

```diff
@@ -41,6 +41,7 @@
         "no_ammo_sound": "a chk!"
       }
     ],
+    "special_when_hit": [ "RETURN_FIRE", 100 ],
```

Comment:
Based on the slide I posted, this should be a 'counter sniper' function co-opted to blindfire at heard targets.
It shouldn't need to have been hit to attempt this.

Reply:
Detecting "firing in your direction" mechanics could be an upgrade to current state, but as it requires much more coding, I think it could be done in subsequent PRs.